x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -0,0 +1,315 @@
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'linux.advanced.artifacts.user.public_key',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'linux.advanced.artifacts.user.interval',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'linux.advanced.elasticsearch.delay',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'linux.advanced.elasticsearch.tls.verify_peer',
+    first_supported_version: '7.11',
+    documentation: 'default is true',
+  },
+  {
+    key: 'linux.advanced.elasticsearch.tls.verify_hostname',
+    first_supported_version: '7.11',
+    documentation: 'default is true',
+  },
+  {
+    key: 'linux.advanced.elasticsearch.tls.ca_cert',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.agent.connection_delay',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.global.base_url',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.global.manifest_relative_url',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.global.ca_cert',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.global.public_key',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.global.interval',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.user.base_url',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.user.ca_cert',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.user.public_key',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.artifacts.user.interval',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.elasticsearch.delay',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.elasticsearch.tls.verify_peer',
+    first_supported_version: '7.11',
+    documentation: 'default is true',
+  },
+  {
+    key: 'mac.advanced.elasticsearch.tls.verify_hostname',
+    first_supported_version: '7.11',
+    documentation: 'default is true',
+  },
+  {
+    key: 'mac.advanced.elasticsearch.tls.ca_cert',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.malware.quarantine',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.kernel.connect',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
+  {
+    key: 'mac.advanced.kernel.harden',
+    first_supported_version: '7.11',
+    documentation: '',
+  },
```

Comment:
@bjmcnic @willyu-elastic do we want this (`mac.advanced.kernel.harden`) to remain the advanced option or should we make it something like `mac.advanced.kernel.harden.enabled` so we can add other options nested under `harden`?

Reply:
I would prefer to move it to `mac.advanced.harden.enabled`